docs/openapi/oad/amphora.oad.yaml

```diff
@@ -0,0 +1,856 @@
+openapi: 3.1.0
+info:
+  title: Amphora Service API
+  description: |
+    This API allows to interact with the Amphora service. It provides functionality to store, fetch and manage secret
+    shared data as used for MPC-based distributed computation on encrypted data in the Carbyne Stack platform.
+
+    The endpoints provided by Amphora are divided into three categories:
+    - User facing: \
+      Used by clients for secret sharing, accessing, and managing secret shares and their tags (*/input-masks*,
+      */masked-input*, */secret-shares*).
+    - Intra VCP (`/intra-vcp`): \
+      Used by services running in the same Virtual Cloud Provider to access locally stored secret shares, e.g. for
+      ingesting into a computation.
+    - Inter VCP (`/inter-vcp`): \
+      Used for communication with other Virtual Cloud Providers in the same Virtual Cloud.
+  version: 0.1.0
+  license:
+    name: Apache-2.0
+    url: https://github.com/carbynestack/amphora/blob/0b1866f9fbe1619729fc16fd8f9c7af679eaf979/LICENSE
+  contact:
+    name: Carbyne Stack
+    url: https://carbynestack.io/
+tags:
+  - name: User facing
```

Comment:
I think it should be "User-facing", "Inter-VCP", "Intra-VCP"

Reply:
Changed